src/udp_server.cpp

```diff
@@ -116,6 +119,11 @@ void UDPServer::handleReceive(const boost::system::error_code& error, std::size_
 void UDPServer::handleSend(std::vector<uint8_t> sent_payload,
   const boost::system::error_code& ec, std::size_t bytes_transferred)
 {
+  // unused
```

Comment:
Here is another way to do it:
`
void UDPServer::handleSend(__attribute__((unused)) std::vector<uint8_t> sent_payload,
  __attribute__((unused)) const boost::system::error_code& ec, __attribute__((unused)) std::size_t bytes_transferred){
}
`

Reply:
Thanks, good to know, that only works with GCC right?

Reply:
Both GCC and CLANG